### PR TITLE
Googletrans

### PR DESCRIPTION
**Pull request by SekouD**
Added recipe for googletrans.
https://github.com/ssut/py-googletrans
https://pypi.org/project/googletrans/

### CODE REVIEW

File: recipes/googletrans/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{% set hash_value = "49cce281c5ded86fc1ef0429e240da650bae2dd7939dc5e664d80c33fa8ba3b0" %}
+
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://files.pythonhosted.org/packages/49/0f/eac53560e99a6b1e3b3e18e8b98f0e0041bfc4f1ee1e1b74cdf5530786d0/googletrans-2.3.0.tar.gz
```
Comment (synapticarbors) [inline recipes/googletrans/meta.yaml]: Is it possible to use the canonical url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

File: recipes/googletrans/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - requests
+    - pytest
+  run:
+    - python
+    - requests
+
+test:
+  imports:
+    - googletrans
+  commands:
```
Comment (synapticarbors) [inline recipes/googletrans/meta.yaml]: You're not actually running any tests other than that `tox` exists, so I think you can drop this and the `test/requires` section.

File: recipes/googletrans/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch_python: True
+  preserve_egg_dir: True
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/googletrans/meta.yaml]: I think you may only need `python`, `setuptools` and `pip` in the `host` section.

File: recipes/googletrans/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch_python: True
```
Comment (isuruf) [inline recipes/googletrans/meta.yaml]: Use `noarch: python` instead of this. You can do this in the feedstock and rerender.

**Pull request merged**